README.md

```diff
@@ -1,24 +1,64 @@
-# README
+# Choreganize
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is the "backend" repository for the Chorganize application. It is responsible for sending api responses to our [frontend application](https://github.com/choreorganize/choreganize).
 
-Things you may want to cover:
+### About this Project
 
-* Ruby version
+Chorganize is an application in which users can create, complete, and monitor chores for their household. Each chore is weighted by difficulty to ensure fairness across all the users of a household.
 
-* System dependencies
+### Database Schema
+<!-- upload image here -->
 
-* Configuration
+### Getting Started
 
-* Database creation
+These instructions will give you a copy of the project up and running on
+your local machine for development and testing purposes.
 
-* Database initialization
+## Local Setup
 
-* How to run the test suite
+1. Fork and Clone the repo
+2. Install gem packages: `bundle install`
+3. Setup the database: `rails db:create`
+4. Run migrations: ` rails db:migrate`
+5. Install the Figaro gem: `bundle exec figaro install`
+6. Create an account with [Open Weather Map](https://home.openweathermap.org/users/sign_up) and request an API key
+7. Add your API key to the `application.yml` created by Figaro:
+  ```rb
+  OPEN_WEATHER_KEY: your_api_key
+  ```
 
-* Services (job queues, cache servers, search engines, etc.)
+## Versions
 
-* Deployment instructions
+- Ruby 2.7.2
 
-* ...
+- Rails 5.2.6
+
+## Important Gems
+Testing
+* [rspec-rails](https://github.com/rspec/rspec-rails)
+* [capybara](https://github.com/teamcapybara/capybara)
+* [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers)
+* [simplecov](https://github.com/simplecov-ruby/simplecov)
+* [webmock](https://github.com/bblimke/webmock)
+* [vcr](https://github.com/vcr/vcr)
+* [Bootstrap](https://github.com/twbs/bootstrap-rubygem)
+
+API Consumption
```

Comment:
Can we add in 'json' and 'jsonapi-serializer`?

Reply:
Good catch! The deed is done.